services/core-web/src/components/modalContent/NoticeOfDepartureModal.tsx

```diff
@@ -319,7 +305,8 @@ const NoticeOfDepartureModal: React.FC<NoticeOfDepartureModalProps> = (props) =>
         </div>
         <FieldArray
           name="nod_contacts"
-          component={(componentProps) => renderContacts(componentProps, disabled)}
+          component={NodContacts}
+          disabled={disabled}
```

Comment:
Also this was causing issues. The `(componentProps) => renderContacts` bit was causing constant re-renders. Made renderContacts/NodContacts into an actual FC